docs/topics/rest-level-0-mission-design-tradeoffs.adoc

```diff
@@ -3,9 +3,13 @@
 |====================================================================
 |Pros           |Cons
 a|
-* Fast prototyping
-* Flexible API Design
-* HTTP endpoints allow clients to be language agnostic
+* This booster enables fast prototyping.
+* The API Design is flexible.
+* HTTP endpoints allow clients to be language agnostic.
 a|
-* As an application or service matures, the {name-mission-http-api} approach may not scale well to properly support a clean API design or use cases involving database interactions. Any operations involving shared, mutable state must be integrated with an appropriate backing datastore. All requests handled by an API designed in this manner will be scoped only to the container servicing the request. Therefore there is no guarantee that subsequent requests will be served by the same container.
-|====================================================================
\ No newline at end of file
+* As an application or service matures, the {name-mission-http-api} approach may not scale well. It may not
```

Comment:
Replace _may_ with _can_ or _might_ (_may_ is only used with permissions as per the IBM Style Guide).